src/uu/ls/src/ls.rs

```diff
@@ -2925,42 +2934,38 @@ fn display_item_long(
             }
         };
 
-        write!(
-            output_display,
-            "{}{} {}",
-            format_args!("{leading_char}?????????"),
-            if item.security_context.len() > 1 {
-                // GNU `ls` uses a "." character to indicate a file with a security context,
-                // but not other alternate access method.
-                "."
-            } else {
-                ""
-            },
-            pad_left("?", padding.link_count)
-        )
-        .unwrap();
+        output_display.extend(
+            format_args!("{leading_char}?????????")
+                .to_string()
```

Comment:
This is identical to `format!("{leading_char}?????????")`

Reply:
Correct, I can simplify further.

Reply:
Done.